lib/HeroBonus.cpp

```diff
@@ -888,10 +899,12 @@ void CBonusSystemNode::getBonusesRec(BonusList &out, const CSelector &selector,
 void CBonusSystemNode::getAllBonusesRec(BonusList &out) const
 {
 	BonusList beforeUpdate;
-	FOREACH_CPARENT(p)

```

Comment:
Function above looks obsolete, it should be removed along with FOREACH_CPARENT macro or changed accordingly to this one.

Reply:
Done